src/clients/wol.rb

```diff
@@ -140,11 +142,11 @@ def main
           @overview = WOL.Overview
           UI.ChangeWidget(Id(:table), :Items, @overview)
         elsif @ret == :wake
-          @mac = Convert.to_string(UI.QueryWidget(Id(:table), :CurrentItem))
-          if @mac != nil && @mac != ""
-            @cmd = Builtins.sformat("/usr/bin/wol %1", @mac)
-            Popup.ShowFeedback(_("Waking remote host"), @mac)
-            SCR.Execute(path(".target.bash"), @cmd)
+          mac = Convert.to_string(UI.QueryWidget(Id(:table), :CurrentItem))
+          if mac != nil && mac != ""
+            cmd = Builtins.sformat("/usr/bin/wol %1", mac.shellescape)
+            Popup.ShowFeedback(_("Waking remote host"), mac)
+            SCR.Execute(path(".target.bash"), cmd)
```

Comment:
NP: Why not starting using `Yast::Execute` instead?

Reply:
I would like, but I would like to avoid potential bugs as I do not have good machine for testing WOL. This can show popup or can raise exception so I would like to do minimal changes unless I have good way to test it or if code is well covered by tests. Neither is true for this module.